pool/test/test_utils.ml

```diff
@@ -115,15 +115,21 @@ module Model = struct
       }
   ;;
 
-  let create_contact ?id ?name ?(with_terms_accepted = true) () =
+  let create_contact
+    ?id
+    ?(language = Pool_common.Language.En)
```

Comment:
Is this default value expected? Should be possible to directly pass the optional argument to the record as a contact shouldn't get a language by default.